backend/resume_evaluations/router.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import APIRouter
+
+from .schemas import ResumeEvaluationRequest, ResumeEvaluationResponse
+from .service import evaluate_resume
+
+router = APIRouter(
+    prefix="/resume_evaluations",
+    tags=["resume_evaluations"],
+)
+
+
+@router.post("/evaluate", response_model=ResumeEvaluationResponse)  # noqa: FAST001
```

Comment:
Remove `response_model` param and `# noqa: FAST001` comment

```suggestion
@router.post("/evaluate")
```